lib/membrane_rtc_engine/endpoints/webrtc/rtp_munger.ex

```diff
@@ -96,6 +96,15 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.RTPMunger do
   @spec generate_padding_packet(t(), Track.t()) :: {t(), Membrane.Buffer.t() | nil}
   def generate_padding_packet(rtp_munger, track) when rtp_munger.last_marker do
     # We can only generate padding packets at frame boundary
+
+    # convert from 1/s to 1/ms
+    clock_rate = div(rtp_munger.clock_rate, 1000)
+    now = System.monotonic_time(:millisecond)
+
+    # estimated time elapsed from the last packet arrival, in clock rate
+    # We add it to last_timestamp to avoid increasing the jitter value calculated by the client
+    timestamp_offset = div(now - rtp_munger.last_packet_arrival, clock_rate)
```

Comment:
Changes in this file are from another PR and we haven't confirmed yet that we want to merge it

Reply:
:thinking: 
![image](https://user-images.githubusercontent.com/20843107/216237771-1f1d8d9e-edd6-4409-b0e2-599b501a90db.png)